src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+// Actions
+const ADD = 'bookstore/books/add';
+const DEL = 'bookstore/books/del';
+
+// Empty array of books
+const initialState = [];
+
+// Reducer
+export default function booksReducer(state = initialState, action = {}) {
+  switch (action.type) {
+    case ADD:
+      return [...state, {
+        ...action.payload,
+      }];
+    case DEL:
+      return state.filter((book) => book.id !== action.id);
+    default: return state;
+  }
```

Comment:
- [ ] Kindly make sure to `payload` which is the property holding the `id`.

   _**Suggestion:**_ ` return state.filter((id) => book.id !== action.payload.id);`